recipes/libavif/all/conanfile.py

```diff
@@ -35,52 +37,71 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        try:
+            del self.settings.compiler.libcxx
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.cppstd
+        except Exception:
+            pass
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     @property
     def _has_dav1d(self):
         return self.options.with_decoder == "dav1d"
 
     def requirements(self):
-        self.requires("libaom-av1/3.1.2")
+        self.requires("libaom-av1/3.4.0")
         self.requires("libyuv/cci.20201106")
```

Comment:
By the way, cci versioning of `libyuv` shouldn't have been accepted in https://github.com/conan-io/conan-center-index/pull/3488. There are libyuv versions (there is no tag, but version is tracked upstream: https://chromium.googlesource.com/libyuv/libyuv/+/f71c83552d373f0ff41833b17e2880632d8561d7/include/libyuv/version.h).

Reply:
see https://github.com/conan-io/conan-center-index/pull/13010